### PR TITLE
Update Samsung certificate login UI

### PR DESCRIPTION
**Pull request by abhishek-kp1**
### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

File: docs/application/dotnet/get-started/certificates/creating-certificates.md [hunk 1]
```diff
@@ -72,10 +72,18 @@ It is used to create an author signature, which ensures the integrity of the app
 
 3. Sign in to Samsung account
 
+    A pop-up notification appear, indicating a 5-minute timeout for accessing Samsung account sign-in page.
```
Comment (safir-srbd) [inline docs/application/dotnet/get-started/certificates/creating-certificates.md:75]:
```suggestion
    A pop-up notification appears, indicating a 5-minute timeout for accessing the Samsung account sign-in page.
```

**Review by safir-srbd — COMMENTED** [review]
@abhishek-kp1 Hello. Hope you are doing well.

After reviewing the PR, I have suggested one short change. Please implement it.

File: docs/application/dotnet/get-started/certificates/creating-certificates.md (hunk 1 above)
Comment (abhishek-kp1) [reply docs/application/dotnet/get-started/certificates/creating-certificates.md]: changes done. Thank you.

**Review by safir-srbd — APPROVED** [review]
@abhishek-kp1 Thank you for implementing the suggested changes. I have also reviewed the new images you have added, and they look good to me.

I am approving this PR for merging.

**Pull request merged**